helm/external-dns-app/files/serviceaccount.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
this is a duplicated object with almost the same content than the original serviceaccount.yaml, which might cause maintenance issues in the future. (editing only one leading to discrepancies depending on the provider)

Reply:
This isn't intended to be a permanent solution. We plan on implementing this in upstream [amazon-eks-pod-identity-webhook](https://github.com/aws/amazon-eks-pod-identity-webhook), but we don't have the capacity for it now. At the moment we want to get rid of Kiam and this was the only way we could do it without introducing yet another value in the `cluster-values` created by `cluster-apps-operator`

Reply:
As mentioned on slack, we are not convinced this is a better approach than using values.

It's also worth mentioning that this resource won't be part of the helm lifecycle, which means that deletion/renaming/etc won't happen.